content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_kubeconfig_user.md

```diff
@@ -1,3 +1,16 @@
+<!--
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference conent, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+
+Output a kubeconfig file for an additional user
```

Comment:
taking a look again here, the kubeadm_alpha_kubeconfig command is deprecated.
prior art indicated that the generator skips Cobra commands that have the Deprecated field, so not sure why it's included here again. :thinking: 

i expected kubeadm_alpha to be empty for 1.21.

not a blocker, but just wanted to mention this.